myTrip/tests/unittest/trip/test.py

```diff
@@ -0,0 +1,55 @@
+"""Testing module for checkpoint views"""
+
+import json
+from trip.models import Trip
+from datetime import datetime
+from registration.models import CustomUser
+from django.test import TestCase, Client
+
+class ViewTest(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.client = Client()
+        user = CustomUser(email = 'ln@gmail.com', password = 'root')
+        user.save()
+        trip = Trip(user = user, title = 'title', description = 'description', status = 0)
+        trip.save()
+
+    def test_get(self):
```

Comment:
organize imports
rename tests for better understanding, the test name should be self-descriptive
add docstrings
"One test should only test one thing" - L.Halamaha

check this out: https://github.com/Lv-246Python/myTrip/blob/1e009c5af925530d3fd4f4ad513f9e7af96d6760/myTrip/tests/unittests/registration/test_models.py